salt/addons/create_namespace.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+KUBECTL_PATH="${KUBECTL_PATH:-/usr/bin/kubectl}"
+
+# dir where manifests are
+DIR=/etc/kubernetes/addons
+
+############################################################################
+
+abort() { echo "FATAL: $@" ; exit 1 ; }
+
+NS_ARGS="--namespace=kube-system"
+SRV_ARGS="--server=http://127.0.0.1:8080"
+
+echo "Creating kube-system namespace..."
+# use kubectl to create kube-system namespace
+NAMESPACE=`eval "$KUBECTL_PATH $SRV_ARGS get namespaces | grep kube-system | cat"`
+if [ ! "$NAMESPACE" ]; then
+	$KUBECTL_PATH $SRV_ARGS create -f "$DIR/namespace.yaml"
```

Comment:
And could we just move this command to a `cmd.run` in the `sls` file? It doesn't make much sense to invoke a script with `cmd.script` just for running a command...